tests/commands_test.py

```diff
@@ -299,6 +299,9 @@ async def test_incr_errors(mcache: Client) -> None:
     with pytest.raises(ClientException):
         await mcache.incr(key, 3.14)  # type: ignore[arg-type]
 
+    value = await mcache.incr(b'not:' + key, 5)
+    assert value is None
```

Comment:
Fixing typing errors.

```suggestion
    ivalue = await mcache.incr(b'not:' + key, 5)
    assert ivalue is None
```